queue_entry.py

```diff
@@ -26,7 +26,6 @@
 import time
 import queue_model_objects_v1 as queue_model_objects
 import os
-import ShapeHistory as shape_history
```

Comment:
@JieNanMAXIV are you going to push another commit to fix this to this PR? So I can merge it